src/plugins/rhbz.c

```diff
@@ -208,6 +208,21 @@ bool rhbz_login(struct abrt_xmlrpc *ax, const char *login, const char *password)
     return true;
 }
 
+
+bool add_session_api_key(struct abrt_xmlrpc *ax, const char *api_key)
```

Comment:
This should have the `rhbz_` prefix to be consistent with the rest of the module.

Reply:
Good catch. I will fix that.

Reply:
Updated.